src/opendr/perception/object_detection_2d/detr/dependencies.ini

```diff
@@ -0,0 +1,14 @@
+[runtime]
+# 'python' key expects a value using the Python requirements file format
+#  https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format 
+python=torch>=1.5.0
+       torchvision>=0.6.0
+       git+https://github.com/cocodataset/cocoapi.git#subdirectory=PythonAPI&egg=pycocotools
```

Comment:
`cocoapi` is also already used by the Pose Estimation methods.
If possible we should use only a single version otherwise we will get this kind of errors:
```
ERROR: Cannot install pycocotools 2.0 (from git+https://github.com/cidl-auth/cocoapi#subdirectory=PythonAPI) and pycocotools 2.0 (from git+https://github.com/cocodataset/cocoapi.git#subdirectory=PythonAPI&egg=pycocotools) because these package versions have conflicting dependencies.
```

Reply:
I missed that, I updated the dependencies.ini such that it is now the same version. On my computer the test is running without errors, so I hope it is fixed now.

Reply:
No changes have been applied yet to this branch and I still get the exact same error when running the test.

Reply:
I think I fixed this error, I believe it was due to a circular import. However, the API CI test failed, but I haven't found out yet why that is. I will try to found out what the source is.

> A couple of comments still have to be addressed.
> 
> Then I checked again the tests and I still get errors:
> 
> ```
> (venv) stefi@libeccio:~/develop/opendr_internal$ python3 -m unittest discover -s tests/sources/tools/perception/object_detection_2d/detr/
> E
> ======================================================================
> ERROR: test_detr (unittest.loader._FailedTest)
> ----------------------------------------------------------------------
> ImportError: Failed to import test module: test_detr
> Traceback (most recent call last):
>   File "/usr/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
>     module = self._get_module_from_name(name)
>   File "/usr/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
>     __import__(name)
>   File "/home/stefi/develop/opendr_internal/tests/sources/tools/perception/object_detection_2d/detr/test_detr.py", line 23, in <module>
>     from opendr.perception.object_detection_2d.detr.detr_learner import DetrLearner
>   File "/home/stefi/develop/opendr_internal/src/opendr/perception/object_detection_2d/detr/detr_learner.py", line 30, in <module>
>     from opendr.perception.object_detection_2d.detr.algorithm.datasets import build_dataset, get_coco_api_from_dataset
>   File "/home/stefi/develop/opendr_internal/src/opendr/perception/object_detection_2d/detr/algorithm/datasets/__init__.py", line 20, in <module>
>     from .coco import build as build_coco
>   File "/home/stefi/develop/opendr_internal/src/opendr/perception/object_detection_2d/detr/algorithm/datasets/coco.py", line 28, in <module>
>     import opendr.perception.object_detection_2d.detr.algorithm.datasets.transforms as T
> AttributeError: module 'opendr.perception.object_detection_2d.detr.algorithm' has no attribute 'datasets'
> 
> 
> ----------------------------------------------------------------------
> Ran 1 test in 0.000s
> 
> FAILED (errors=1)
> ```